app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -120,6 +122,12 @@ object TelemetryWrapper {
         val WHATS_NEW = "whats_new"
         val RESUME = "resume"
         val RELOAD = "refresh"
+        val SSLDataInvalidError = "SSLDateInvalidError"
```

Comment:
I don't think we should add these all manually - either the toString method of SslError gives us something useful (which I'm pretty sure is the case), or we just return the code and handle it on the telemetry end.

Reply:
Actually, on second thought, if the toString method doesn't return anything useful, then we should have this int => string handled in TelemetryWrapper. These don't need to be variables because we don't expect to use them anywhere else, so if we NEED to because the API isn't returning a useful string result for the errors, add a function like sslErrorCodeToString that switches through the code and returns a string.